### PR TITLE
TS strict all components/types starting with letter 'A'

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/accordion/stories/Accordion.stories.tsx [hunk 1]
```diff
@@ -20,53 +20,55 @@ type ItemType = {
...
+    ]
+  },
+  render: (args) => (
+    <Accordion {...args}>
+      {(item) => <Item key={(item as ItemType).key} title={(item as ItemType).title}>{(item as ItemType).key}</Item>}
+    </Accordion>
+  )
+};
+
+export const DefaultExpandedKeys: AccordionStory = {
```
Comment (ktabors) [inline packages/@react-spectrum/accordion/stories/Accordion.stories.tsx:46]: default-expanded-keys, disabled-keys, and disabled-default-expanded-keys are not rendering in storybook.

Comment (snowystinger) [reply packages/@react-spectrum/accordion/stories/Accordion.stories.tsx:46]: ah, i think i see, i wasn't providing a render function. don't know why that worked locally

**Review by ktabors — DISMISSED** [review]

**Review by reidbarber — APPROVED** [review]

**Review by ktabors — APPROVED** [review]
still looks good

**Review by reidbarber — APPROVED** [review]
LGTM

**Pull request merged**